src/tools/rbd/action/Ggate.cc

```diff
@@ -161,10 +108,7 @@ int execute_map(const po::variables_map &vm,
   }
 
   if (vm.count("options")) {
-    r = parse_options(vm["options"].as<std::vector<std::string>>(), &args);
-    if (r < 0) {
-      return r;
-    }
+    utils::append_options_as_args(vm["options"].as<std::vector<std::string>>(), &args);
```

Comment:
over 80 columns (here and everywhere else `append_options_as_args` is called)
```suggestion
    utils::append_options_as_args(vm["options"].as<std::vector<std::string>>(),
                                  &args);
```